.pre-commit-config.yaml

```
@@ -0,0 +1,6 @@
+repos:
+-   repo: https://github.com/psf/black
```

yamllint (1.35.1)

[warning] too many spaces after hyphen

(hyphens)